<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addComputerDialog</class>
 <widget class="QDialog" name="addComputerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_computer_header">
     <property name="text">
      <string>&lt;h2&gt;Add computer&lt;/h2&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_computer_name">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_computer_name"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_computer_type">
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_computer_type_electronic">
        <property name="text">
         <string>Electronic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_computer_type_transistor">
        <property name="text">
         <string>Transistor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_computer_type_mechatronic">
        <property name="text">
         <string>Mechatronic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_computer_type_other">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_computer_built">
     <property name="title">
      <string>Was the computer built?</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButton_computer_yes">
        <property name="text">
         <string>Yes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_computer_no">
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_computer_year_built">
        <property name="text">
         <string>If yes, what year was it built?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_computer_year_built">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_add_computer">
     <property name="text">
      <string>Add Computer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
